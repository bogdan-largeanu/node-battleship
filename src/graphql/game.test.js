describe('GraphQL Type: Game', () => {
    it(`fetch game's only scalar values`, async () => {
        const { data } = await query(
            {
                query: `
                {
                    game(id: 1) {
                        id
                    }
                }`
            }
        );

        expect(data).toMatchSnapshot();
    });

    it(`fetch game's battlefield's scalar values`, async () => {
        const { data } = await query(
            {
                query: `
                {
                    game(id: 1) {
                        battlefields {
                            id
                        }
                    }
                }`
            }
        );

        expect(data).toMatchSnapshot();
    });
});
