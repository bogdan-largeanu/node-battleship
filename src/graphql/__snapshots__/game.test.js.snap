// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GraphQL Type: Game fetch game's battlefield's scalar values 1`] = `
Object {
  "game": Object {
    "battlefields": Array [
      Object {
        "id": "1",
      },
      Object {
        "id": "2",
      },
    ],
  },
}
`;

exports[`GraphQL Type: Game fetch game's only scalar values 1`] = `
Object {
  "game": Object {
    "id": "1",
  },
}
`;
